t/reconfigure/consensus.t.js

```diff
@@ -74,12 +73,15 @@ function prove (async, assert) {
                 consensus.watch(abend)
                 setTimeout(async(), 2500)
             }, function () {
-                wait = async()
```

Comment:
You should still wait. Use the wait to make sure that you get your listener test to fire.